auditlog/receivers.py

```
@@ -99,36 +100,52 @@ def log_access(sender, instance, **kwargs):
 def _create_log_entry(
     action, instance, sender, diff_old, diff_new, fields_to_check=None, force_log=False
 ):
-    pre_log_results = pre_log.send(
-        sender,
-        instance=instance,
-        action=action,
-    )
+    pre_log_results = None
     error = None
+    logger = logging.getLogger(__name__)
+
+    try:
+        pre_log_results = pre_log.send(
```

are we sure that signals can throw errors? im pretty sure they fail silently.

---

right, it wasn't in the initial code either so i just wanted to be extra safe here, wasn't sure